java/vector/src/main/java/org/apache/arrow/vector/ipc/ArrowFileReader.java

```diff
@@ -112,6 +113,16 @@ public void initialize() throws IOException {
     }
   }
 
+  /**
+   * Get custom metadata.
+   */
+  public Map<String, String> getMetaData() {
+    if (footer != null) {
+      return footer.getMetaData();
+    }
+    return null;
```

Comment:
is this consistent with other methods, I think returning an empty map might be better?

Reply:
ok, fixed.